src/Controller/ProfilerController.php

```diff
@@ -83,6 +83,39 @@ public function explainAction($token, $connectionName, $query)
         ]));
     }
 
+    /**
+     * Renders the profiler panel for the given token.
+     *
+     * @param string $token          The profiler token
+     * @param string $connectionName
+     * @param int    $query
+     *
+     * @return Response A Response instance
+     */
+    public function runAction($token, $connectionName, $query)
```

Comment:
This is old school PHP 5 style code, we don't use it for new stuff
```suggestion
    public function runAction(string $token, string $connectionName, int $query): Response
```